src/frontend/hooks/useTracking.ts

```diff
@@ -2,11 +2,10 @@ import * as Location from 'expo-location';
 import * as TaskManager from 'expo-task-manager';
 import {useCallback, useState} from 'react';
 import {usePersistedTrack} from './persistedState/usePersistedTrack';
-import React from 'react';
 import {FullLocationData} from '../sharedTypes/location';
-import {useGPSModalContext} from '../contexts/GPSModalContext';
-
-export const LOCATION_TASK_NAME = 'background-location-task';
+import {useInterval} from 'react-native-confirmation-code-field/esm/useTimer';
```

Comment:
For reviewer: this import may be unstable, let me know if we should create our own useInterval in utils